chia/simulator/block_tools.py

```diff
@@ -712,9 +798,14 @@ def get_consecutive_blocks(
         include_transactions: bool = False,
         skip_overflow: bool = False,
         min_signage_point: int = -1,
+        block_fillrate: Optional[int] = None,
```

Comment:
an alternative would be to make `include_transactions` be an integer, 0=no transactions, 1=5 transactions per block, 2=full blocks